README.md

```diff
@@ -97,20 +98,16 @@ Here are the steps to using this package:
 
 ### Configuration
 
-At the moment things are not very well exposed or configurable without going into a few top-level files and configuring them. If this package has more interest, I expect it to become more configurable over time.
-Here is where different configuration lives:
+`YATM` relies on several environment variables that need to be set.
```

Comment:
It might be worth mentioning direnv as a nice way to set this up.

https://direnv.net/

Reply:
[ff6b187](https://github.com/audrow/yatm/pull/404/commits/ff6b187cf123180d7f986e686a1e7c593c386098)